src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -45,7 +46,7 @@ public void delete(@PathVariable Long id) {
     }
 
     @PutMapping("/{id}")
-    public BookDto updateBook(@PathVariable Long id, @RequestBody BookDto bookDto) {
+    public BookDto updateBook(@PathVariable Long id, @RequestBody @Valid BookDto bookDto) {
```

Comment:
```suggestion
    public BookDto updateBook(@PathVariable @Positive Long id, @RequestBody @Valid BookDto bookDto) {
```

Please, add `@Validated` under the controller. Read about `@Positive and @Validated` annotations